calcom/cal.com
# feat: email undefined value

## What does this PR do?
This PR fixes a bug that the undefined value should not be rendered inside the email template.

Fixes #9238
Fixes #7829

**Environment**: Production

## Type of change
- Bug fix (non-breaking change which fixes an issue)

## How should this be tested?
- Book a Meeting
- Leave the custom questions empty
- Check your email confirmation and you'll see the undefined as the result
